src/games/strategy/triplea/ai/proAI/data/ProOtherMoveOptions.java

```diff
@@ -36,7 +36,11 @@ public ProTerritory getMax(final Territory t) {
   }
 
   public List<ProTerritory> getAll(final Territory t) {
-    return moveMaps.get(t);
+    final List<ProTerritory> result = moveMaps.get(t);
+    if (result != null) {
+      return result;
+    }
+    return new ArrayList<ProTerritory>();
```

Comment:
Prefer either:
 return new ArrayList<>();
or (guava):
 return Lists.newArrayList();

Makes refactoring easier in the future, one less thing to change.